contrib/platform/src/com/sun/jna/platform/WindowUtils.java

```
@@ -1285,17 +1285,30 @@ public String getProcessFilePath(final HWND hwnd) {
 
             final HANDLE process = Kernel32.INSTANCE.OpenProcess(WinNT.PROCESS_QUERY_INFORMATION | WinNT.PROCESS_VM_READ,
                                                                  false, pid.getValue());
-            if (process == null
-                && Kernel32.INSTANCE.GetLastError() != WinNT.ERROR_ACCESS_DENIED) {
-                throw new Win32Exception(Kernel32.INSTANCE.GetLastError());
+            if (process == null) {
+                if(Kernel32.INSTANCE.GetLastError() != WinNT.ERROR_ACCESS_DENIED) {
+                    throw new Win32Exception(Kernel32.INSTANCE.GetLastError());
+                } else {
+                    // Ignore windows, that can't be accessed
```

This seems to swallow any other unexpected error!